src/components/EmployeeTable/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import EmployeeRow from "./EmployeeTableRow";
+import styles from "../../styles/users.module.css";
```

Comment:
@Doyin02 move the styles you need to this component css.module file, then reference them. 

So it would then be:
```suggestion
import styles from "./EmployeeTable.module.css";
```